src/docfx/Errors.cs

```diff
@@ -764,5 +764,11 @@ public static Error LiquidNotFound(SourceInfo<string?> source)
             public static Error MustacheNotFound(string templateFileName)
                 => new Error(ErrorLevel.Error, "mustache-not-found", $"Mustache template is not found at '{templateFileName}'.");
         }
+
+        public static class SourceMap
+        {
+            public static Error DuplicateContent(string key, IEnumerable<PathString> originalFiles)
+                => new Error(ErrorLevel.Warning, "duplicate-content", $"'{key}' is duplicated from {StringUtility.Join(originalFiles)}");
```

Comment:
Do we want to add a hint that which file is ignored?

Reply:
Do not think this matters, since the user has the clear action indication already.
Please refer to `duplicate-uid`, which has the same error message.

Reply:
`duplicate-content` seems a broad concept, consider `duplicate-sourcemap-item`